templates/nvidia-GPU/nvidiaGPU.cfg

```diff
@@ -9,11 +9,10 @@ PATROL_CONFIG
 "/SENTRY/X/nvidiaGPU/macroList" = { REPLACE = "NVSMI_PATH;/usr/bin/;1" },
 "/SENTRY/X/nvidiaGPU/messageList" = { REPLACE = "" },
 "/SENTRY/X/nvidiaGPU/parameterList" = { REPLACE = "CollectionErrorCount;1" },
-"/SENTRY/X/nvidiaGPU/parent" = { REPLACE = "" },
 "/SENTRY/X/nvidiaGPU/productVersion" = { REPLACE = "10000" },
 "/SENTRY/X/nvidiaGPU/runAlertForEachCollectionError" = { REPLACE = "1" },
 "/SENTRY/X/nvidiaGPU/shareCookies" = { REPLACE = "1" },
-"/SENTRY/X/nvidiaGPU/systemType" = { REPLACE = "other" },
+"/SENTRY/X/nvidiaGPU/systemType" = { REPLACE = "windows" },
```

Comment:
all platform please

Reply:
~~agreed, this should be "other" (all platforms)~~
Already updated, cool! :-)